packages/server/lib/controllers/webhook.controller.ts

```diff
@@ -9,6 +9,12 @@ class WebhookController {
             if (!environmentUuid || !providerConfigKey) {
                 return;
             }
+            const isGloballyDisabled = await featureFlags.isEnabled('external-webhooks', 'global', true, true);
```

Comment:
should the const be called `isGloballyEnabled`?

Reply:
yup, thanks https://github.com/NangoHQ/nango/pull/1487/commits/8c341d6375872304a08adb7c7cfee2a2e9614eab